contracts/satelites/PullPayment.sol

```diff
@@ -23,6 +23,10 @@ contract PullPayment is Ownable {
 
   mapping(address => Payment) internal payments;
 
+  modifier whenNotPaused () {
+    require (!ControllerInterface(owner).paused());
```

Comment:
is it common to have spacing between `require` and `(` in solidity? i always think of js notation, maybe i'm wrong